packages/mdc-fab/_mixins.scss

```diff
@@ -305,6 +312,17 @@ $mdc-fab-icon-enter-duration_: 180ms;
   }
 }
 
+@mixin mdc-fab--touch_($query: mdc-feature-all()) {
```

Comment:
Can we move styles from this mixin and apply directly to `.mdc-fab--touch` css selector?